app/docs/async-api.yaml

```diff
@@ -0,0 +1,80 @@
+asyncapi: 2.4.0
+info:
+  title: FFC Payment Statement Constructor
+  version: '1.0.0'
+  description: Construct data for Payment Statements
+channels:
+  return/message:
+    subscribe:
+      message:
+        $ref: '#/components/messages/ReturnMessage'
+components:
+  messages:
+    ReturnMessage:
+      name: Return Message
+      contentType: application/json
+      payload:
+        $ref: '#/components/schemas/ReturnMessage'
+  schemas:
+    ReturnMessage:
+      type: object
+      required:
+      - sourceSystem
+      - invoiceNumber
+      - frn
+      - currency
+      - value
+      - settlementDate
+      - reference 
+      - settled
```

Comment:
missing detail property

Reply:
added detail as tbc as cant find any example of what will be in there and the payload is an empty string